client/blocks/importer/wordpress/upgrade-plan/with-migration-sticker.tsx

```diff
@@ -20,7 +24,13 @@ const withMigrationSticker =
 			if ( 0 !== siteId ) {
 				addMigrationSticker( siteId );
 			}
-		}, [ addMigrationSticker, siteId ] );
+
+			return () => {
+				if ( 0 !== siteId ) {
+					deleteMigrationSticker( siteId );
```

Comment:
Just a double-check with @merkushin, we won't need the sticker in the checkout step for anything, right?

Reply:
We need it until the end of the checkout.

Reply:
The "Upgrade" step does not get unmounted when `Checkout` opens as it opens in a modal. So the sticker won't get unset when we open the checkout.

Reply:
TIL! I thought it was a different step! Thank you for the answer! =)